Assignment 8/Ques4.js

```diff
@@ -0,0 +1,16 @@
+let myString = 'hello world';
+if (myString.length > 0) {
+    console.log('The length of the string is: ' + myString.length);
+} else {
+    console.log('The string is empty');
+}
+
+let myNumber = parseInt(myString);
```

Comment:
This might break if the string is something like '124samarth' `because parseInt checks whether the first argument of the string is a number or a string. If it's a number, it will return a number; if it's a string, it will return NaN. `

> Try using Number.isInteger or Number.isFinite if you want to check explicitly whether the given value is a number or not.

Reply:
okay i got it